### ISSUE TITLE
Feature Request: Allow disable freezing due date replacement

### ISSUE THREAD

**Issue by gthbusrr** [grocy/grocy#1880]
Currently, freezing and thawing will replace due date with **today + X days**. However, I've a habbit to put chocolates to the freezer before enjoying it (it's weird, I know), but I want it to keep its original due date. Is there a way to achieve it? Thanks~~

**Comment by berrnd** [comment]
Of course, just set the "Default due days after freezing/thawing" to `0` (which is also the default for new products).

Original due date + 0 days = original due date.

> I've a habbit to put chocolates to the freezer before enjoying it

Practically idea / how I would to it personally: Don't track the move to the freezer in grocy for that habbit.

**Comment by berrnd** [comment]
> just set the "Default due days after freezing/thawing" (product option) to `0`

Sorry, I was wrong about that. That's currently only handled for thawing in that manner, not on freezing.

So let's somehow unify that and make it also possible for freezing.
